### PR TITLE
Latest Sentiment analysis UI design (#302)

### PR DESCRIPTION
**Pull request by SteveRosam**
* Update with latest design changes

* Remove -output for compatibility with other projects

* deploy settings for sentiment ui

* fix url prefix

* rename source dir to src

* Library.json source folder

* get env vars instead of hard coded

* Build TFL UI

* update working locally

### CODE REVIEW

File: nodejs/advanced/Sentiment-demo-ui/src/app/services/quix.service.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  /*~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-*/
+  /*WORKING LOCALLY? UPDATE THESE!*/
+  private workingLocally = false; // set to true if working locally
+  private token: string = ""; // Create a token in the Tokens menu and paste it here
+  public workspaceId: string = ''; // Look in the URL for the Quix Portal your workspace ID is after 'workspace='
+  public messagesTopic: string = ''; // get topic name from the Topics page
+  public sentimentTopic: string = ''; // get topic name from the Topics page
+  /*~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-*/
+
+  readonly subdomain = 'dev'; // leave as 'platform'
```
Comment (SteveRosam) [inline nodejs/advanced/Sentiment-demo-ui/src/app/services/quix.service.ts:23]: this needs to be set to `platform` before release

Comment (SteveRosam) [reply nodejs/advanced/Sentiment-demo-ui/src/app/services/quix.service.ts]: done

File: nodejs/advanced/Demo.Data.Stream.Processing/library.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "libraryItemId": "a1c46790-6e68-4d7a-8ce2-18ae414b6bf7",
-    "name": "Streaming Demo - UI",
+    "name": "Streaming demo UI",
```
Comment (tbedford) [inline nodejs/advanced/Demo.Data.Stream.Processing/library.json:3]: > "name": "Streaming demo UI",

Didn't we decide to call it `Sentiment Demo UI`? Or are we referring to something else here? If something else, just want to check it should be in this PR?

**Review by tbedford — APPROVED** [review]
One minor comment otherwise looks good.

File: nodejs/advanced/Demo.Data.Stream.Processing/library.json (hunk 2 above)
Comment (SteveRosam) [reply nodejs/advanced/Demo.Data.Stream.Processing/library.json:3]: This is the name that will show in the library

File: nodejs/advanced/Sentiment-demo-ui/tsconfig.app.json [hunk 3]
```diff
@@ -1,14 +1,15 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```
Comment (tbedford) [inline nodejs/advanced/Sentiment-demo-ui/tsconfig.app.json:1]: Is this kind of comment valid in a json file?

File: nodejs/advanced/Sentiment-demo-ui/tsconfig.spec.json [hunk 4]
```diff
@@ -1,18 +1,18 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```
Comment (tbedford) [inline nodejs/advanced/Sentiment-demo-ui/tsconfig.spec.json:1]: Valid comment?

**Review by tbedford — APPROVED** [review]
Couple of minor comments, otherwise looks OK.

File: nodejs/advanced/Sentiment-demo-ui/tsconfig.spec.json (hunk 4 above)
Comment (SteveRosam) [reply nodejs/advanced/Sentiment-demo-ui/tsconfig.spec.json:1]: i think so. i didnt add this it was mostly auto generated by something

File: nodejs/advanced/Sentiment-demo-ui/tsconfig.app.json (hunk 3 above)
Comment (SteveRosam) [reply nodejs/advanced/Sentiment-demo-ui/tsconfig.app.json:1]: think so. autogenerated file.

**Pull request merged**